app/javascript/chat/chat.jsx

```diff
@@ -1409,7 +1410,35 @@ export default class Chat extends Component {
       .getElementById('jumpback_button')
       .classList.remove('chatchanneljumpback__hide');
   };
-
+  handleImageDrop = (e) => {
+    document.getElementById('messagelist').style.opacity = 1;
```

Comment:
Instead of using getElementById, leverage (P)React paradigms and consider using the [useRef](https://reactjs.org/docs/hooks-reference.html#useref) hook. You can see an example of this in action in the Forem code base here. 👇🏻 https://github.com/forem/forem/blob/2f09256713d6da36619ff761e887ec01687d8c60/app/javascript/packs/flagUserModal.jsx#L100.

Reply:
Tried implementing it, giving some problems in case of hover. I think Creatref is not necessary here.
```
There are a few good use cases for refs:

    Managing focus, text selection, or media playback.
    Triggering imperative animations.
    Integrating with third-party DOM libraries.

Avoid using refs for anything that can be done declaratively.
```
Source: [React Docs](https://reactjs.org/docs/refs-and-the-dom.html)